README.md

```diff
@@ -89,6 +89,8 @@ To use this library you need to ensure you are using the correct version of Reac
 | `maximumValue` | Initial maximum value of the slider.<br/>Default value is 1. | number | No | |
 | `minimumTrackTintColor` | The color used for the track to the left of the button.<br/>Overrides the default blue gradient image on iOS. | [color](https://reactnative.dev/docs/colors) | No | |
 | `minimumValue` | Initial minimum value of the slider.<br/>Default value is 0. | number | No | |
+| `lowerLimit` | Slide lower limit. The user won't be able to slide below this limit. | number | No | |
+| `upperLimit` | Slide upper limit. The user won't be able to slide above this limit. | number | No | |
```

Comment:
There's no Windows implemented for this feature yet, so as the Windows will not be supported let's add `iOS, Android` to the `Platform` column for both `lowerLimit` and `upperLimit`